app/src/main/java/com/ikarabulut/Board.java

```diff
@@ -96,6 +103,8 @@ private Cell topLeftNeighbor(int rowNum, int colNum) {
             topLeftNeighbor = getCell(rows - 1, columns - 1);
         } else if (rowNum == 0) {
             topLeftNeighbor = getCell(rows - 1, colNum - 1);
+        } else if (colNum == 0) {
```

Comment:
Was this a bug or something that was caught during manual testing?  Is there an automated test to ensure it doesn't come back?

Reply:
Yes, it was a bug caught when I was iterating through the whole world to evolve the generation. Ill add a test.